src/pages/index.js

```diff
@@ -31,11 +36,15 @@ export async function getStaticProps(context) {
 
   const websiteData = await fireDb.getWebsiteData(targetedHackathon)
 
-  const { FeatureFlags } = websiteData
+  const { featureFlags, BuildConfig } = websiteData
```

Comment:
Just out of curiosity, what would be the reason to make such a change?

Reply:
Hmm never mind! I guess it's just the way it got passed in.

Reply:
The problem with FeatureFlags is that CMS would have problems LOL. So unless we update cms, this is it. What's your opinion?